src/icons/general/IconSecurityClassification.tsx

```diff
@@ -5,31 +5,15 @@ import AccessibleSVG from '../accessible-svg';
 import { bdlGray } from '../../styles/variables';
 import { Icon } from '../iconTypes';
 
-const IconSecurityClassification = ({
-    className = '',
-    height = 32,
-    color = bdlGray,
-    title,
-    strokeWidth = 2,
-    width = 32,
-}: Icon) => {
+const IconSecurityClassification = ({ className = '', height = 12, color = bdlGray, title, width = 12 }: Icon) => {
```

Comment:
can we put a comment here to eventually move to the asset repo version of this? Its a small bit of refactor work so OK to come back to

is it this icon more or less? http://opensource.box.com/box-ui-elements/storybook/?path=/story/icon-line-shield16--shield-16